### ISSUE TITLE
Incorrect ticker symbols from CoinMarketCap

### ISSUE THREAD

**Issue by guopengliang** [ccxt/ccxt#734]
Run `fetchTickers()` from `coinmarketcap` and we get incorrect ticker symbols:
```js
{
    // These are fine (100 tickers)
    'BTC/USD': { ... },
    'BCH/USD': { ... },
    ...
    // These are NOT correct (1220 tickers)
    'enjin-coin/USD': { ... }, // should be 'ENJ/USD'
    'lampix/USD': { ... }, // should be 'PIX/USD'
    ...
}
```

**Comment by guopengliang** [comment]
I did a little bit research and found that it's because the default `fetchMarkets()` for `coinmarketcap` did not fetched all markets available.

Specifying `{ limit: 0 }` would fix the problem:
https://github.com/ccxt/ccxt/blob/d2b08a6327b77fdcbb46312f0a5cd58c656bfa98/js/coinmarketcap.js#L71
Reference: https://coinmarketcap.com/api/

**Comment by munrocket** [comment]
Fixed this problem in my pull request #553.

**Comment by kroitor** [comment]
This was fixed in one of the previous versions of CCXT, which version of the library do you have?
